mono/profiler/proflog.c

```diff
@@ -173,7 +174,12 @@ static MonoLinkedListSet profiler_thread_list;
  * [flags: 4 bytes] file format flags, should be 0 for now
  * [pid: 4 bytes] pid of the profiled process
  * [port: 2 bytes] tcp port for server if != 0
- * [sysid: 2 bytes] operating system and architecture identifier
+ * [args size: 1 byte] size of args
```

Comment:
Args should be leb as it's possible to come up with more than 256 bytes of arguments.